README.md

```diff
@@ -1,4 +1,4 @@
-SyliusGridBundle [![Build status...](https://secure.travis-ci.org/Sylius/SyliusGridBundle.png?branch=master)](http://travis-ci.org/Sylius/SyliusGridBundle)
+SyliusGridBundle
```

Comment:
But would be perfect to replace it with github actions badge
```suggestion
SyliusGridBundle [![Build](https://github.com/Sylius/SyliusGridBundle/actions/workflows/build.yml/badge.svg)](https://github.com/Sylius/SyliusGridBundle/actions/workflows/build.yml)
```

Reply:
@lchrusciel You're right, that's even better.

Reply:
I've added the badge 👌🏻